README.md

```diff
@@ -65,7 +65,55 @@ A [tenant repository](https://github.com/fluxcd/flux2-multi-tenancy/tree/dev-tea
     └── podinfo-values.yaml
 ```
 
-## Onboard tenants
+## Get this example up and running
+
+**Pre-reqs:**
+- GitHub Account and a [PAT]
+- git
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)
+- [flux cli](https://fluxcd.io/docs/installation/)
+
+**1. Fork this repository**
+
+Fork this repository on your personal GitHub account.
+
+**2. Clone your version of this repository**
+
+```sh
+git clone https://github.com/YOUR_ACCOUNT/flux2-multi-tenancy
+```
+
+**3. Check flux cli version**
+
+Verify that your flux cli is `v0.26` or above:
+
+```sh
+flux --version
+```
+
+**4. Bootstrap production cluster**
+
+The command below will create a new `kind` cluster named `flux2-mt-prod` and bootstrap flux into it.
+
+```sh
+GITHUB_USER=YOUR_ACCOUNT make bootstrap-prod
```

Comment:
I'm not sure if we should hide the Flux command behind Make. I think it's better to have the command here inline so that Windows users can follow the guide.

Reply:
Agreed - completely removed it now.